AwsCryptographicMaterialProviders/dafny/AwsCryptographyKeyStore/src/KMSKeystoreOperations.dfy

```diff
@@ -341,6 +407,7 @@ module {:options "/functionSyntax:4" } KMSKeystoreOperations {
       res.Success?
       ==>
         && |kmsClient.History.Encrypt| == |old(kmsClient.History.Encrypt)| + 1
+        && kmsClient.History.GenerateDataKey == old(kmsClient.History.GenerateDataKey)
```

Comment:
I think I know this problem; but doing this should not be required.
I am going to see if I can fix this.